internal/armador/chart_test.go

```diff
@@ -41,8 +49,9 @@ func TestChart_parseArmadorFile(t *testing.T) {
 			} else if err == nil && tt.wantErr {
 				t.Errorf("expected error, no error received for %s", tt.name)
 			}
-			if !reflect.DeepEqual(tt.chart, tt.expected) {
-				t.Errorf("%s test failed. \nGot: %v \nExpected: %v", tt.name, tt.chart, tt.expected)
+			if len(deep.Equal(tt.chart, tt.expected)) > 0 {
+				logger.GetLogger().Warnf("Diff %v", deep.Equal(tt.chart, tt.expected))
```

Comment:
remove the logging from the test. instead of using `len` check if the diff is nil, and error the right info on failure